### DISCUSSION TITLE
Build for aarch64 Mac configuring .ini file wrong

### DISCUSSION THREAD

**Discussion by jmenzies12** [eclipse-tycho/tycho#2085] [General]
Hey there,

We're using tycho to build our eclipse application. I've just added the config to build for aarch64 mac, this works in that we are now downloading the correct jars for that build, however our ini file doesn't correctly get changed to point to the right equinox launcher, despite it being in the jar. 

Apologies if this is the wrong place to ask, I wasn't entirely sure if it's tycho or pde that does this.

Using Tycho 1.3.0 if that helps.

Cheers
Jonathan

**Comment by laeubi** [comment]
> Using Tycho 1.3.0 if that helps.

Tycho 1.3 is heavily outdated, please upgrade to 3.0.1 first and see if that solves your issue.

**Reply by jmenzies12** [reply]
Hey, thanks for the reply. I tried using 3.0.0 but it just tells me it cannot run with Java 11. What is the most recently supported version of tycho for Java 11?

**Reply by laeubi** [reply]
Tycho requires Java 17 but you can still compile code for 11.
